test/src/e2e_vm_tests/README.md

```diff
@@ -0,0 +1,67 @@
+# Running end-to-end VM tests
+
+This document assumes you have `fuel-core` running on the default port.
+
+## Running all tests
+
+In a second terminal, `cd` into the `sway` repo and run:
+
+```sh
+cargo run --bin test
+```
+
+After the test suite runs, you should see:
+
+```console
+Tests passed.
+_n_ tests run (0 skipped)
+```
+
+## Running specific tests
+
+From the `sway` directory run:
+
+```sh
+cargo run --bin=test -- specific_tests_pattern
```

Comment:
Inconsistent with the `--bin=test` vs `--bin test` above.  Both work though...

Reply:
Good catch! Fixed